app/Helpers/MiscHelper.php

```diff
@@ -2,20 +2,20 @@
 
 namespace App\Helpers;
 
-class MiscHelper{
+class MiscHelper {
 
-    public static function array_diff_assoc_recursive($array1, $array2){
-        $difference=array();
-        foreach($array1 as $key => $value) {
-            if( is_array($value) ) {
-                if( !isset($array2[$key]) || !is_array($array2[$key]) ) {
+    public static function array_diff_assoc_recursive($array1, $array2) {
```

Comment:
__[code_sniffer]__
* Method name "MiscHelper::array_diff_assoc_recursive" is not in camel caps format
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->